src/init/templates/wazuh-server-debian.init

```diff
@@ -19,10 +19,10 @@
 ### END INIT INFO
 
 WAZUH_HOME=WAZUH_HOME_TMP
-WAZUH_CONTROL="$WAZUH_HOMEbin/wazuh-control"
+WAZUH_CONTROL="${WAZUH_HOME}bin/wazuh-server"
 
 start() {
-    ${WAZUH_CONTROL} start
+    ${WAZUH_CONTROL} start -r
```

Comment:
We should add a comment to remove this once the engine does not create resources in root paths, to avoid forgetting about this and executing the server always with root privileges

```suggestion
    # TODO: Remove root flag
    ${WAZUH_SERVER} start -r
```